package/MDAnalysis/lib/util.py

```diff
@@ -30,11 +30,7 @@
 ---------------------
 
 .. autofunction:: filename
-.. function:: openany(directory[,mode='r'])
-
-   Context manager to open a compressed (bzip2, gzip) or plain file
-   (uses :func:`anyopen`).
-
+.. autofunction:: openany
```

Comment:
When I look at the code I see
       @contextmanager
       def openany():

(Sorry difficult to link to code on mobile gh interface.)

If the docs look fine then it's not a problem, of course.